src/scripts.js

```diff
@@ -42,7 +54,7 @@ import './images/stopwatch.svg'
 import './images/trophy.svg'
 
 
-
+function all() {
```

Comment:
@MattTuring We'll research whether this is the best way to go about this, but I'm glad it's working for now.